src/UserSpaceInstrumentation/TrampolineTest.cpp

```diff
@@ -709,8 +709,9 @@ TEST_F(InstrumentFunctionTest, DoSomething) {
       pid_, function_address_, function_code_, trampoline_address_, entry_payload_function_address_,
       return_trampoline_address_, capstone_handle_, relocation_map_);
   EXPECT_THAT(address_after_prologue_or_error, HasNoError());
-  ErrorMessageOr<void> result = InstrumentFunction(
-      pid_, function_address_, address_after_prologue_or_error.value(), trampoline_address_);
+  ErrorMessageOr<void> result =
+      InstrumentFunction(pid_, function_address_, /*function_id=*/0,
```

Comment:
0 is default value of function id in the trampoline - maybe use 42?

Reply:
done